libsolidity/Version.cpp

```diff
@@ -20,7 +20,7 @@
  * Versioning.
  */
 
-#include <libsolidity/Version.h>
+#include <Version.h>
```

Comment:
`Version.h` might not be unique, so I guess it would be better to include it with full path.